groupie/src/main/java/com/genius/groupie/ExpandableGroup.java

```diff
@@ -64,4 +64,71 @@ public void onToggleExpanded() {
             notifyItemRangeInserted(oldSize, newSize - oldSize);
         }
     }
+    
+    private boolean dispatchChildChanges(Group group) {
```

Comment:
A little late to the party, but another possible implementation would be to set/clear the group data observer on the children depending on whether isExpanded is true/false. Then all of the following code that checks if child change dispatches should fire could be deleted.